recipes/annotated-types/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "annotated-types" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/annotated_types-{{ version }}.tar.gz
+  sha256: fae0ca44ad43b7e54dbb02b22137885b1cb1cb7a312fe9fe7a6d77a0e5b0443a
+
+build:
+  skip: true  # [py<37]
```

Since this is pure Python, we can make this noarch:
```suggestion
  noarch: python
```

---

Yup, good call (I just didn't look closely enough at the grayskull output)